angular/src/app/landing/data-files/data-files.component.html

```diff
@@ -1,5 +1,5 @@
 <!-- This bloack is for the data access session of landing page -->
-<div id="dataAccess">
+<div *ngIf="inBrowser" id="dataAccess">
```

Comment:
I think this use of `inBrowser` is unnecessary here as there is already one in place in line 52.  It unnecessarily removes content from view which will actually be noticeable by the user when the metadata includes a large file list.  

The use on line 52 guards against the use of the primeng tree component that does not work on the server side.  It also includes an "else" block which displays the "Loading file list..." which be seen for a short time in the browser when there is a large number of files.

Reply:
I've removed this `inBrowser` (c92bd94) and tested on oardev.

Reply:
Agree.